Makefile

```diff
@@ -40,6 +42,12 @@ dev-image: build
 image:
 	docker build -t $(IMAGE) .
 
+push-container: image
+	gcloud auth configure-docker
+	docker push $(IMAGE)
+
+push: push-container
```

Comment:
why `push` and `push-container`?

Reply:
I copied most of the container image push bits of the `Makefile` from [node-problem-detector](https://github.com/kubernetes/node-problem-detector/blob/master/Makefile). It looks like they have an extra step to also build a tar file with the built binaries and push to GCS bucket `gs://kubernetes-release`. This functionality could be added in the future to the descheduler too.

Let me know if you want this changed.